Directory.Packages.props

```diff
@@ -4,6 +4,7 @@
 
```

Comment:
I just checked, and it's already merging `*.dll`: https://github.com/NuGet/NuGet.Client/blob/40648c3cb3de37cf62b194accdc8e7117b36245f/src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.csproj#L101

It doesn't need to though. .NET (Core) runtime already ships the DLL (and in fact, it won't be int he publish output, since I conditioned the package on netfx and netstandard). On .NET Framework, msbuild already ships the dll, so if it wasn't found in the merged assembly, the assembly loader should still find it in msbuild's directory.